__test__/FetchLoader.test.js

```diff
@@ -20,36 +20,70 @@ describe(`FetchLoader`, () => {
     fetchMock.restore()
   })
 
+  const props = {
+    host: `foo`,
+    resource: `bar`
+  }
+
   test(`until the fetch promise is not resolved a loading message is displayed, then goes away`, async () => {
     fetchMock.get(`*`, `[]`)
-    const wrapper = shallow(<ComponentWithFetchLoader host={`foo`} resource={`bar`}/>)
+    const wrapper = shallow(<ComponentWithFetchLoader {...props} />)
+
+    expect(wrapper.find(`#loading-message`)).toHaveLength(1)
+    expect(wrapper.find(CalloutAlert)).toHaveLength(0)
 
-    expect(wrapper.find(`#loading-message`).exists()).toBe(true)
-    expect(wrapper.find(CalloutAlert).exists()).toBe(false)
     await wrapper.instance().componentDidMount()
     wrapper.update()
-    expect(wrapper.find(`#loading-message`).exists()).toBe(false)
-    expect(wrapper.find(CalloutAlert).exists()).toBe(false)
+
+    expect(wrapper.find(`#loading-message`)).toHaveLength(0)
```

Comment:
One thing I noticed is that we should remove IDs from HTML and use classes instead. Could you change that now that you’re at it? Mostly I’m concerned with more than one fetch loader in one page. It would be a transient inconsistency, but...